Docs/source/install/hpc/lumi.rst

```diff
@@ -43,21 +44,40 @@ We recommend to store the above lines in a file, such as ``$HOME/lumi_warpx.prof
 
    source $HOME/lumi_warpx.profile
 
+And since LUMI does not yet provide a module for them, install c-blosc and ADIOS2:
+
+.. code-block:: bash
+
+   export CMAKE_PREFIX_PATH=${HOME}/sw/lumi/gpu/c-blosc-1.21.1:$CMAKE_PREFIX_PATH
+   export CMAKE_PREFIX_PATH=${HOME}/sw/lumi/gpu/adios2-2.8.3:$CMAKE_PREFIX_PATH
+
+   # c-blosc (I/O compression)
+   git clone -b v1.21.1 https://github.com/Blosc/c-blosc.git src/c-blosc
+   rm -rf src/c-blosc-build
+   cmake -S src/c-blosc -B src/c-blosc-build -DBUILD_TESTS=OFF -DBUILD_BENCHMARKS=OFF -DDEACTIVATE_AVX2=OFF -DCMAKE_INSTALL_PREFIX=${HOME}/sw/lumi/gpu/c-blosc-1.21.1
+   cmake --build src/c-blosc-build --target install --parallel 16
+
+   # ADIOS2
+   git clone -b v2.8.3 https://github.com/ornladios/ADIOS2.git src/adios2
+   rm -rf src/adios2-build
+   cmake -S src/adios2 -B src/adios2-build -DADIOS2_USE_Blosc=ON -DADIOS2_USE_Fortran=OFF -DADIOS2_USE_Python=OFF -DADIOS2_USE_ZeroMQ=OFF -DCMAKE_INSTALL_PREFIX=${HOME}/sw/lumi/gpu//adios2-2.8.3
+   cmake --build src/adios2-build --target install -j 16
+
 Then, ``cd`` into the directory ``$HOME/src/warpx`` and use the following commands to compile:
 
 .. code-block:: bash
 
    cd $HOME/src/warpx
    rm -rf build
 
-   cmake -S . -B build -DWarpX_DIMS=3 -DWarpX_COMPUTE=HIP -DWarpX_PSATD=ON
-   cmake --build build -j 6
+   cmake -S . -B build -DWarpX_COMPUTE=HIP
+   cmake --build build -j 32
```

Comment:
Just checking: how many cores per head node?
To be a friendly user, maybe use 1/8 or 1/4th of physical cores max, since login nodes are a shared resource :)

Reply:
The login nodes are AMD EPYC 7742 64-Core processors, so maybe we can use 16 threads for this...